Lint-Whoppers/nutshell
# Mj event calendar

# Description

added events form

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1) git fetch --all
2) git checkout mj-eventCalendar
3)  What to check for events form with name, date and location


# Checklist:
